time-tracker-webapi/src/TimeTracker.Api/Views/Shared/_Layout.cshtml

```diff
@@ -7,18 +7,50 @@
 <html>
 <head>
     <title>@ViewBag.Title</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <environment include="Development">
+        <link href="/node_modules/bootstrap/dist/css/bootstrap.css" rel="stylesheet"/>
```

Comment:
Not sure here, but wouldn't it be better to include a CDN version rather than self-hosted? @cocowmn can maybe comment on this better than I can 😄

Reply:
I guess it depends on the needs. CDN comes with a handful of benefits:
* You don't have to host the code yourself
* If a client has visited other websites that use content from the same CDN, that content will be cached and will load from disk rather than making a network request, which will give the user a perceived faster page load.

It can be easier to self-host sometimes, especially when you're working with a lot of packages and when those packages contain inter-dependencies. I'd suggest trying to do a CDN, then if you hit hurdles and it's sucking up time, bail and default back to self hosting.